homeassistant/components/ccm15/coordinator.py

```diff
@@ -0,0 +1,86 @@
+"""Climate device for CCM15 coordinator."""
+import datetime
+import logging
+
+from ccm15 import CCM15Device, CCM15DeviceState, CCM15SlaveDevice
+import httpx
+
+from homeassistant.components.climate import HVACMode
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import (
+    CONST_FAN_CMD_MAP,
+    CONST_STATE_CMD_MAP,
+    DEFAULT_INTERVAL,
+    DEFAULT_TIMEOUT,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class CCM15Coordinator(DataUpdateCoordinator[CCM15DeviceState]):
+    """Class to coordinate multiple CCM15Climate devices."""
+
+    def __init__(self, hass: HomeAssistant, host: str, port: int) -> None:
+        """Initialize the coordinator."""
+        super().__init__(
+            hass,
+            _LOGGER,
+            name=host,
+            update_method=self._async_update_data,
```

Comment:
```suggestion
```

Not needed

Reply:
No? I'm in the bus, I'll try to give a link when I'm home. Feel free to remind ne

Reply:
Please check if the call to super requires the method reference or not.

Reply:
Well, the default implmentation of `_async_update_data` literally is:
```py
    async def _async_update_data(self) -> _DataT:
        """Fetch the latest data from the source."""
        if self.update_method is None:
            raise NotImplementedError("Update method not implemented")
        return await self.update_method()
```
So you can just omit it

Reply:
I stand corrected.